src/result.js

```diff
@@ -46,6 +47,24 @@ class Result {
   }
 
   /**
+   * Returns the serialized representation of the HEAD.
+   *
+   * @returns {String} serialized version of the DOM HEAD
+  */
+  getHead() {
```

Comment:
I prefer explicit API for getting private things from a class. getters/setters kind of make me think it can be misused :). But that is probably just me. Keeping this explicit API. 

Or I can provide a more verborse API like `domContents` which would  return an object like:
```javascript
{
   'head': head,
   'body': body
}
```

would you prefer something like that?